dev/watch.js

```diff
@@ -96,6 +97,9 @@ chokidar.watch(`${sassDir}/**/*.scss`).on('change', changedFile => {
                 filesToCompile.map(file => file.replace('scss', 'css'))
             );
         })
+        .then(() => {
+            copyFiles.task();
```

Comment:
it would be good to avoid coupling the code that needs to run to the runner's task definition (this will break if the runner no longer defines Things To Do as `{task: x, description: y}`). 

i might be better to provide an external stylesheet in dev, as we do with the `.head` stuff? that way there's nothing special that need to happen with reloading?